### PR TITLE
Cleanup security tests

### PR DESCRIPTION
**Pull request by mcculls**
Drop NexusTestSupport from security tests in favour of a simpler setup (first step towards removing NexusTestSupport completely)

http://bamboo.s/browse/NX3-OSSF161-1 :white_check_mark:

### CODE REVIEW

File: components/nexus-security/src/test/java/org/sonatype/nexus/security/user/EmptyRoleManagementTest.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 import org.junit.Test;
 
+import static org.junit.Assert.assertEquals;
```
Comment (cstamas) [inline components/nexus-security/src/test/java/org/sonatype/nexus/security/user/EmptyRoleManagementTest.java]: unused import?

**Comment by cstamas** [comment]
+1

File: components/nexus-security/src/test/java/org/sonatype/nexus/security/user/EmptyRoleManagementTest.java (hunk 1 above)
Comment (mcculls) [inline components/nexus-security/src/test/java/org/sonatype/nexus/security/user/EmptyRoleManagementTest.java]: The old `NexusTestSupport` class defines its own `assertEquals` so the process of removing it involved importing the appropriate JUnit method (or switching to the equivalent in hamcrest)

**Comment by mcculls** [comment]
Merged as 8e5e303

**Pull request closed without merging**